CMakeLists.txt

```diff
@@ -1,5 +1,11 @@
 cmake_minimum_required(VERSION 3.2)
 
+include("cmake/HunterGate.cmake")
```

Comment:
Please split the changes so it will be easier to adapt them in the future.

See:
- https://docs.hunter.sh/en/latest/creating-new/patch.html
```
Perfect hunterization should contain only:

    - Adding HunterGate module
    - Including it with some URL/SHA1
    - Adding hunter_add_package commands
```